sections/advanced.md

```diff
@@ -228,6 +228,14 @@ If you do not want to have hooks installed by default on the stage passed
 during a `pre-commit install --hook-type ...`, please set the [`default_stages`](#top_level-default_stages)
 top level configuration property to the desired stages, also as an array.
 
+Hooks can also confine themselves to a set of stages through their [`stages`](#hooks-stages)
```

Comment:
the sentence at 222 already says essentially this

Reply:
That sentence talks about user config in `.pre-commit-config.yaml`, while this added one is about the hook "definition" config in `.pre-commit-hooks.yaml`.

Reply:
the settings work the same independent of where they are defined

Reply:
Combined the paragraphs in e9c320aa20c6e261f648373bc285af1b62b773c9